src/Tizen.Multimedia.Remoting/MediaController/MediaControlPlaylist.cs

```diff
@@ -0,0 +1,120 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using Native = Interop.MediaControllerClient;
+
+namespace Tizen.Multimedia.Remoting
+{
+    /// <summary>
+    /// Represents playlist for media control.
+    /// </summary>
+    /// <since_tizen> 5 </since_tizen>
+    public abstract class Playlist
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Playlist"/> class.
+        /// </summary>
+        /// <since_tizen> 5 </since_tizen>
+        protected Playlist() { }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Playlist"/> class.
+        /// </summary>
+        /// <since_tizen> 5 </since_tizen>
+        protected Playlist(string name, string index, ulong position)
+        {
+            Index = index ?? throw new ArgumentNullException("Playlist index is not valid to convert number.");
+            if (!Int32.TryParse(index, out int result))
+            {
+                throw new ArgumentException("Playlist index is not vaild to convert to number.");
+            }
+
+            Name = name ?? throw new ArgumentNullException("Playlist name is not set.");
+            Position = position;
+        }
+
+        /// <summary>
+        /// Gets or sets the index of the media in the playist.
+        /// </summary>
+        /// <since_tizen> 5 </since_tizen>
+        public string Index { get; protected set; }
+
+        /// <summary>
+        /// Gets or sets the name of playist.
+        /// </summary>
+        /// <since_tizen> 5 </since_tizen>
+        public string Name { get; protected set; }
+
+        /// <summary>
+        /// Gets or sets the playing position of the media indicated by <see cref="Index"/>.
+        /// </summary>
+        /// <since_tizen> 5 </since_tizen>
+        public ulong Position { get; protected set; }
+
+        /// <summary>
+        /// Gets the total number of media in the playlist.
+        /// </summary>
+        /// <since_tizen> 5 </since_tizen>
+        public virtual int TotalCount { get; } = 1;
+
+        /// <summary>
+        /// Creates an object of the MediaControlPlaylist with the specified name, index, position.
+        /// </summary>
+        /// <param name="name"></param>
+        /// <param name="index"></param>
+        /// <param name="position"></param>
+        /// <returns>A <see cref="MediaControlSimplePlaylist"/> instance.</returns>
+        /// <exception cref="ArgumentException"><paramref name="index"/> cannot be converted to number.</exception>
+        /// <exception cref="ArgumentNullException">
+        /// <paramref name="name"/> or <paramref name="index"/> is not set.
+        /// </exception>
+        /// <since_tizen> 5 </since_tizen>
+        public static MediaControlSimplePlaylist Create(string name, string index, ulong position)
```

Comment:
Why do you make factory method?
Do you have any reason?

Usually, the factory method is used when need to make several subtypes.
and must returing super class type

But it is not

Reply:
If you want to prepare for extend, 
It must returning Playlist instead of MediaControlSimplePlaylist 

``` c#
public static Playlist Create(string name, string index, ulong position);
public static Playlist Create(....);
```